go/database/mpt/node_flusher_test.go

```diff
@@ -44,32 +45,43 @@ func TestNodeFlusher_StartAndStopWithDisabledFlusher(t *testing.T) {
 }
 
 func TestNodeFlusher_TriggersFlushesPeriodically(t *testing.T) {
```

Comment:
Given the new test code, the test should probably be renamed into something like
`TestNodeFlusher_UsesTickerToTriggerFlushes`